backend/config/sync/user-role.admin.json

```diff
@@ -3,6 +3,18 @@
   "description": "Pastor",
   "type": "admin",
   "permissions": [
+    {
+      "action": "api::church-info.church-info.createLocalization"
+    },
+    {
+      "action": "api::church-info.church-info.delete"
```

Comment:
Let's not let admin delete church-info

Reply:
Okay, I will resolve this by deselecting "delete" in Strapi Settings and exporting config sync. When I hit export, does that automatically make updates to my vscode files? Or do I have to pull them again?

Reply:
The best way is by trying out. Check on the file before editing. Check on the file after editing. Check on the file after exporting.

Pulling your code has nothing to do in this case as when you pull code it will get the code from the remote repository. Does your local changes have anything to do with remote?